sdk/storage/azure-storage-file-share/tests/recordings/test_file_client.pyTestStorageFileClienttest_closing_pipeline_client.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Entries": [],
```

Comment:
Got quite a few empty recordings here. Haven't looked at the tests yet but these should all likely not need to be recorded.